### ISSUE TITLE
Unable to install correctly

### ISSUE THREAD

**Issue by vincenzon** [jupyter-widgets/ipywidgets#2633]
I am trying to use ipywidgets but  keep getting the same error:
```
import ipywidgets as widgets
widgets.IntSlider()
```
```
Widget Javascript not detected.  It may not be installed or enabled properly.
Error displaying widget: model not found
```

I am running jupyterlab out of a docker container build using this Dockerfile:

```
FROM ubuntu:latest

RUN apt-get update
RUN DEBIAN_FRONTEND=noninteractive apt-get install -y python3 python3-dev sudo tzdata curl jupyter

RUN pip3 install jupyterlab

RUN curl -sL https://deb.nodesource.com/setup_13.x | sudo -E bash -
RUN apt-get install -y nodejs
RUN pip3 install ipywidgets
RUN jupyter nbextension enable --py widgetsnbextension
RUN jupyter labextension install @jupyter-widgets/jupyterlab-manager

RUN groupadd -g 999 user && useradd -r -u 999 -g user -ms /bin/bash user && usermod -aG sudo user && usermod -u 1000 user
RUN echo "\nuser ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
RUN mkdir /home/user/jupyter; chown user /home/user/jupyter; chmod a+w /home/user/jupyter

USER user
WORKDIR /home/user
CMD ["jupyter-lab",  "--ip='*'", "--no-browser", "--NotebookApp.token=''", "--NotebookApp.password=''"]
```

Is there something wrong with my installation of ipywidgets + jupyter extension?

**Comment by pbugnion** [comment]
Thanks for raising this, and apologies for the late reply.

tldr ; Make sure you install `ipywidgets==0.7.0` by changing `RUN pip3 install ipywidgets` to `RUN pip3 install --upgrade ipywidgets`

-----

Explanation:

Without my suggested fix, your docker image is running `ipywidgets 6.0`. This is very outdated, and is incompatible with jupyterlab-manager 0.1.1, which you install later.

I haven't looked into exactly why your image ends up with ipywidgets 6, but the way you're installing jupyter is somewhat confused:

- you first install it with apt-get, which is likely to be highly outdated
- you then install jupyterlab with pip3

I'd recommend using one of the [supported Jupyter docker images](https://jupyter-docker-stacks.readthedocs.io/en/latest/) instead.

**Comment by pbugnion** [comment]
I've closed this, but feel free to re-open if you still have issues.
